### ISSUE TITLE
`import/extensions` not flagging errors in `export ... from "./mod"`

### ISSUE THREAD

**Issue by isiahmeadows** [benmosher/eslint-plugin-import#964]
As in, it's missing both of these, which should be considered errors:

```js
/* eslint import/extensions: [2, "always"] */
export {foo} from "./foo";

/* eslint import/extensions: [2, "never"] */
export {foo} from "./foo.js";
```
